src/main/java/at/hannibal2/skyhanni/utils/PrimitiveIngredient.kt

```diff
@@ -2,25 +2,27 @@ package at.hannibal2.skyhanni.utils
 
 import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.SKYBLOCK_COIN
 import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.NumberUtil.formatInt
 import io.github.moulberry.notenoughupdates.recipes.Ingredient
 
-class PrimitiveIngredient(val internalName: NEUInternalName, val count: Double = 1.0) {
-
-    constructor(internalName: NEUInternalName, count: Int) : this(internalName, count.toDouble())
+class PrimitiveIngredient(val internalName: NEUInternalName, val count: Int = 1) {
```

Comment:
as talked about elsewhere, this cant be changed to an int because double is used because coins can be represented as a primitive ingredient